packages/devtools_app/test/support/mocks.dart

```diff
@@ -13,7 +13,8 @@ import 'package:devtools_app/src/service_extensions.dart' as extensions;
 import 'package:devtools_app/src/service_manager.dart';
 import 'package:devtools_app/src/stream_value_listenable.dart';
 import 'package:devtools_app/src/memory/memory_controller.dart';
-import 'package:devtools_app/src/memory/flutter/memory_controller.dart' as flutterMemory;
+import 'package:devtools_app/src/memory/flutter/memory_controller.dart'
+    as flutterMemory;
```

Comment:
nit: name import prefixes as flutter_memory rather than flutterMemory 
https://dart.dev/guides/language/effective-dart/style#do-name-import-prefixes-using-lowercase_with_underscores

Reply:
Done.